File: app/domain/entities/article.py
from dataclasses import dataclass
from datetime import datetime


@dataclass
class ArticleEntity:
    unique_username: str
    title: str
    content: str
    user_id: int
    nickname: str
    category: str
    created_at: datetime
    article_id: int
    likes: int
    dislikes: int
    reaction: str | None = None

File: admin-services/internal/domain/article.go
package domain

import "time"

type Article struct {
	Id        int       `json:"article_id"`
	Title     string    `json:"title"`
	UserID    int       `json:"user_id"`
	CreatedAt time.Time `json:"created_at"`
}

type StatArticles struct {
	Value int
	Err   string
}
